packages/nextlove/package.json

```diff
@@ -22,6 +22,7 @@
     "yalc": "npm run build && yalc push"
   },
   "peerDependencies": {
+    "esbuild-register": ">=3",
```

Comment:
if `esbuild-register` is a peer, `esbuild` will have to be too

Reply:
We should be ok, I think peer deps cascade, but now I need to check on that

https://github.com/egoist/esbuild-register/blob/4184f944da7bd6a654389c08e46be6d555da326b/package.json#L40-L42

Reply:
Yes, seems to work as expected
<img width="626" alt="image" src="https://user-images.githubusercontent.com/721372/232136724-80335212-cf9d-4ca1-b234-1bd3767875f4.png">